src/pages/Confirm/index.jsx

```diff
@@ -11,14 +11,15 @@ import { RatingImg } from './Confirm.styled'
 
 const Confirm = () => {
   const { id } = useParams()
+  const packageId = JSON.parse(sessionStorage.getItem('packageId'))
   const history = useHistory()
   const [isFinalModalOpen, toggleFinalModal] = useState(false)
   const [isRatingModalOpen, toggleRatingModal] = useState(false)
   const [rating, setRating] = useState(0)
   const { insertClientRate } = InsertClientRate()
   const { confirmPackage, packageInformation } = useConfirmPackage()
 
-  if (!id) {
+  if (!id || !packageId) {
     history.push('/check')
```

Comment:
Use the function redirectToCheck instead, or is unnecesary?.

I mean the function at line 54

Reply:
it would be the same

> When i return to some page using the package code at the URL, the page is redirecting me to check again. Try entering in:
> 
> https://deploy-preview-163--azordev-dasher-user.netlify.app/chat/test
> 
> I think this doesn't should occurs.
> 
> Some that we can try is to use an object at the localStorage to save the package id
> 
> ```js
> {
>    'test': 'asdag-dfsd-24-dasd',
>    'other': 'cxsad-sfsdf-15-fdsfs'
> }
> ```
> 
> Then when the user enter in a new package, you don't need remove prev packages, and when the user return to some page using the URL you can get the package id from the localStorage instead of do a new call to get the package id.
> 
> The other option is when the user try enter to the page using the package name, and the package id doesn't exists at localStorage, then try search with a new query, if else exists, then redirect to check

this is what should happen. that means that there is no package Id in the session store, maybe we could use local storage instead so it would persist until the process ends and we erased it, what do you think  @Katsu08 @Israel-Laguan

Reply:
we should focus on general functionality. Lets drop the session thing on this PR or leave as it is, @Katsu08 please make an issue to fix persistent package code